network/network_linux.go

```diff
@@ -91,6 +91,11 @@ func (nm *networkManager) newNetworkImpl(nwInfo *NetworkInfo, extIf *externalInt
 	case opModeTransparentVlan:
 		log.Printf("Transparent vlan mode")
 		ifName = extIf.Name
+		// Disable RP filter first
+		_, err := nm.plClient.ExecuteCommand(DisableRPFilterCmd)
+		if err != nil {
+			return nil, fmt.Errorf("[transparent vlan] failed to disable rp filter: %w", err)
+		}
```

Comment:
lets scope this to vnet namespace level.. set disable rp_filter in vnet namespace on its creation